### PR TITLE
Prepare for publish

### PR DESCRIPTION
**Pull request by DJMcNab**
This is the first PR which requires a review.

I would also ask for someone to skim the rest of the repository.

There are a few things I am aware of which need changing:
1) linebender/vello#531 needs backporting
2) We need our standard copyright headers

### CODE REVIEW

**Comment by xStrom** [comment]
I started by looking at parts of the repo and making #5, #6, and #7.

File: README.md [hunk 1]
```diff
@@ -17,6 +17,14 @@ This repository contains two crates for working with Android NDK Tracing.
 The most useful of these is likely to be the higher level Tracing Android Trace,
 which depends on the lower level Android Trace
 
+<figure>
+<img src="https://github.com/linebender/android_trace/assets/36049421/a7f03b74-d690-42be-91b5-326fbb698a03" alt="Screenshot showing a thread timeline including spans of a single thread">
```
Comment (xStrom) [inline README.md:21]:
```suggestion
<img src="https://github.com/linebender/android_trace/assets/36049421/a7f03b74-d690-42be-91b5-326fbb698a03" alt="Screenshot showing a thread timeline including spans of a single thread.">
```

File: README.md [hunk 2]
```diff
@@ -17,6 +17,14 @@ This repository contains two crates for working with Android NDK Tracing.
...
+<figcaption>
+
+Tracing spans for [Vello](https://github.com/linebender/vello) shown in Android GPU Inspector, created using Tracing Android Trace
```
Comment (xStrom) [inline README.md:24]:
```suggestion
Tracing spans for [Vello](https://github.com/linebender/vello) shown in Android GPU Inspector, created using Tracing Android Trace.
```

File: tracing_android_trace/README.md [hunk 3]
```diff
@@ -29,7 +29,7 @@ Note that this does not currently support `tracing` *events*, only spans.
 This limitation is due to the underlying Android platform APIs.
 
 <figure>
-<img src="https://github.com/DJMcNab/android_trace/assets/36049421/a7f03b74-d690-42be-91b5-326fbb698a03" alt="Screenshot showing a thread timeline including spans of a single thread">
+<img src="https://github.com/linebender/android_trace/assets/36049421/a7f03b74-d690-42be-91b5-326fbb698a03" alt="Screenshot showing a thread timeline including spans of a single thread">
```
Comment (xStrom) [inline tracing_android_trace/README.md:32]:
```suggestion
<img src="https://github.com/linebender/android_trace/assets/36049421/a7f03b74-d690-42be-91b5-326fbb698a03" alt="Screenshot showing a thread timeline including spans of a single thread.">
```

File: tracing_android_trace/src/sync_layer.rs [hunk 4]
```diff
@@ -24,7 +24,7 @@ use tracing_subscriber::{
 /// See the [crate level documentation](crate#android-api-levels) for more.
 ///
 /// <figure>
-/// <img src="https://github.com/DJMcNab/android_trace/assets/36049421/a7f03b74-d690-42be-91b5-326fbb698a03" alt="Screenshot showing a thread timeline including spans of a single thread">
+/// <img src="https://github.com/linebender/android_trace/assets/36049421/a7f03b74-d690-42be-91b5-326fbb698a03" alt="Screenshot showing a thread timeline including spans of a single thread">
```
Comment (xStrom) [inline tracing_android_trace/src/sync_layer.rs:27]:
```suggestion
/// <img src="https://github.com/linebender/android_trace/assets/36049421/a7f03b74-d690-42be-91b5-326fbb698a03" alt="Screenshot showing a thread timeline including spans of a single thread.">
```

**Review by xStrom — COMMENTED** [review]
Let's sprinkle some periods on these sentences. For the `alt` attributes especially, as that improves the screen reader experience.

> [End alt-text with punctuation (comma, colon, or period) to indicate a pause in reading, when appropriate.](https://www.w3.org/WAI/GL/techniques.htm)

File: README.md [hunk 5]
```diff
@@ -17,6 +17,14 @@ This repository contains two crates for working with Android NDK Tracing.
 The most useful of these is likely to be the higher level Tracing Android Trace,
 which depends on the lower level Android Trace
```
Comment (DJMcNab) [inline README.md:18]:
```suggestion
which depends on the lower level Android Trace.
```

File: tracing_android_trace/README.md (hunk 3 above)
Comment (xStrom) [reply tracing_android_trace/README.md:32]: Looks like this and the following one were marked as resolved but no corresponding commit was made.

Comment (DJMcNab) [reply tracing_android_trace/README.md:32]: Oops. Not sure how I managed that

Thanks.

**Review by xStrom — APPROVED** [review]

**Pull request merged**